rust/ffi/node/src/lib.rs

```diff
@@ -49,6 +53,33 @@ struct JsTable {
 
 impl Finalize for JsTable {}
 
+// TODO: object_store didn't export this type so I copied it.
+// Make a requiest to object_store to export this type
+#[derive(Debug)]
+pub struct StaticCredentialProvider<T> {
```

Comment:
Why this is in nodejs ffi?

Reply:
This shouldn't I'll make a PR to object store and remove this once object store exports this.